cli/printer/text.go

```diff
@@ -370,14 +365,14 @@ func (t *TextPrinter) PrintTasks(tasks ...*models.Task) error {
 		return strings.Replace(t.DeployID, ".", ":", 1)
 	}
 
-	rows := []string{"TASK ID | TASK NAME | ENVIRONMENT | DEPLOY | SCALE "}
+	rows := []string{"TASK ID | TASK NAME | ENVIRONMENT | DEPLOY | STATUS "}
```

Comment:
This would require documentation where we show the output of a task to be updated. For example, http://layer0.ims.io/guides/one_off_task/ .

Reply:
👍